docs/upgrading/upgrading-to-4.3.md

```diff
@@ -24,4 +24,9 @@ Steps to overwrite key storage:
 
 - Go to the key storage page
 - Select the private key need to be overwritten then click the Overwrite Key button in the toolbar
-- Follow the same steps as [Add a Private Key](/manual/system-configs.html#key-storage) to overwrite the keys
\ No newline at end of file
+- Follow the same steps as [Add a Private Key](/manual/system-configs.html#key-storage) to overwrite the keys
+
+## For Github SCM users:
```

Comment:
Is this specific to the 4.3 release or just timing on the GitHub side.  If the same issue applies to all versions of Rundeck this note needs to go on the SCM Plugin docs page, not the Upgrading notes.

Reply:
This Github issue applies to all versions of Rundeck.

Reply:
if there is no fix for this in 4.3, it doesn't belong here

Reply:
Greg is right.  As I said above, it belongs on the feature page as a warning box.